packages/frontend/src/modules/changePassword/changePassword.component.tsx

```diff
@@ -0,0 +1,142 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import { formStyles, phoneFormStyles, tabletFormStyles,
+  messageStyles, errorMessageStyles, phoneMessageStyles, tabletMessageStyles } from './changePassword.styles';
+import { SubmitHandler, SubmitErrorHandler, useForm } from 'react-hook-form';
+
+import UserService from '~shared/services/user.service';
+import { useUsersSelector } from '~/hooks/useUsersSelector';
+import { validatePassword } from '~shared/services/validation.service';
+
+import FormField from '~shared/components/field/field.component';
+import { ROUTER_KEYS } from '~shared/keys';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  onTablet: boolean,
+  onPhone: boolean,
+};
+
+interface Form {
+  password: string;
+  repeatedPassword: string;
+}
+
+const ChangePassword: React.FunctionComponent<Props> = ({ onTablet, onPhone }) => {
+  const { user, setUser, setIsUserLoading } = useUsersSelector();
+  const userService = new UserService(setUser);

```

Comment:
do not set user in your service, separete logic with fetching data from server and set this data to state